src/i18n/translation_en.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": {
+    "error": {
+      "contact-us": "Please try again later or contact us.",
+      "connection-error": "Oops, an error occurred while trying to connect. $t(domain.error.contact-us)",
+      "unspecified-error": "Oops, an unspecified error occurred. $t(domain.error.contact-us)",
+      "validation-error":  "Oops, an error occurred during validation. $t(domain.error.contact-us)",
+      "gateway-error": "Oops, there is a network error. $t(domain.error.contact-us)"
```

Comment:
You can have gateways that don't use the network. I'd suggest to either introducing a general `network-error` key or changing the message.

Reply:
Yes, I will take a look on it